akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/MultiNodeClusterShardingSpecSpec.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding
+
+import scala.concurrent.duration._
+
+import akka.actor._
+import akka.cluster.MemberStatus
+import akka.testkit._
+import com.typesafe.config.ConfigFactory
+
+abstract class MultiNodeClusterShardingSpecConfig(mode: String, rememberEntities: Boolean)
+    extends MultiNodeClusterShardingConfig(mode, rememberEntities) {
+
+  val first = role("first")
+  val second = role("second")
+  val third = role("third")
+  val fourth = role("fourth")
```

Comment:
I think those should be defined in specific tests, since number of nodes are very dependent on what is tested

Reply:
This is in the test of the base test.

Reply:
The base spec is `MultiNodeClusterShardingSpecConfig`, I got the `*SpecSpec` from the testkit tests. Will update for clarity.